pkg/webhook/stats_reporter.go

```diff
@@ -62,30 +70,33 @@ func newStatsReporter() (StatsReporter, error) {
 	return &reporter{ctx: ctx}, nil
 }
 
-// Captures req count metric, recording the count and the duration
-func (r *reporter) ReportAdmissionRequest(response admissionReqRes, d time.Duration) error {
-	ctx, err := tag.New(
-		r.ctx,
-		tag.Insert(admissionStatusKey, string(response)),
-	)
-	if err != nil {
+func (r *reporter) ReportAdmissionRequest(response requestResponse, d time.Duration) error {
```

Comment:
nit: this looks like it can go away now that it's unused.

Reply:
Done